en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-api-manager.md

```diff
@@ -0,0 +1,297 @@
+    
+- Before starting the upgrading process, make sure that you have read the whole documentation specific to the version upgrade and have a clear understanding of the upgrading process.
+
+- If you have expired certificates in client-truststore, follow the guidelines in [Renewing a CA-Signed Certificate in a Keystore]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/renewing-a-ca-signed-certificate-in-a-keystore/#renewing-a-ca-signed-certificate-in-a-keystore).
+
+- If you have many APIs, there could be a high load on the database during the migration. Hence, increase the database pool size during migration. For more information, see [Tuning JDBC Pool Configurations]({{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/jdbc_tuning/).
+   
+- Start the migration from the lowest environment (e.g., dev) and continue up to the highest before the production (e.g., pre-prod). Run the test cases in the migrated environments to confirm that your functional and non-functional requirements are met in the migrated environment.
+
+- Before you carry out the production migration, run a pilot migration on your pre-prod environment. 
+
+    It will be ideal if the pre-prod environment is similar to the production environment.
+
+    -  If possible, restore a database dump of the production environment to the pre-prod environment and perform the pilot migration.
+
+    -  If the production database dump cannot be used, at least ensure that you have a sufficient amount of data in the database to mimic the production environment.
+    
+- When you follow the above instructions, you can get a rough estimate of the time for the final production update, and you can allocate time slots based on the above analysis. WSO2 recommends that you perform the migration while the system is under minimum traffic. 
+
+- When attempting to migrate a distributed setup, you need to do the data migration for the Control Plane profile only. 
+
+- Disable versioning in the registry configuration when migrating IS as a Key Manager from versions older than WSO2 IS 5.9.0.
+
+    If there are frequently updating registry properties, having the versioning enabled for registry resources in the registry can lead to unnecessary growth in the registry-related tables in the database. To avoid this, versioning has been disabled by default in API Manager 4.1.0.
+    
+    Therefore, if registry versioning was enabled in older versions of WSO2 API-M, it is **required** to run registry version disabling scripts against **the database that is used by the registry**. For example, see step 5 under [Migrating the API Manager configurations]({{base_path}}/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410/#step-1-migrate-the-api-manager-configurations) for the associated scripts.
```

Comment:
The example mentioned in this section may not be valid anymore because this page will be removed with this PR. Therefore, we may have to remove the example.

Reply:
Good catch. Fixed with https://github.com/wso2/docs-apim/pull/6425/commits/6bad3e181fe3a40897a11480348a0b5876c74124